apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -733,53 +733,30 @@ export class DetailViewContents extends React.Component {
               this.props.applicationData.form_data.lastName
             }`}
           </h1>
-          <h4>
-            Meets minimum requirements?{' '}
-            {this.props.applicationData.meets_criteria}
-          </h4>
+          <h4>Meets Guidelines? {this.props.applicationData.meets_criteria}</h4>
           {this.props.applicationData.application_type ===
             ApplicationTypes.teacher && (
             <h4>
               Meets scholarship requirements?{' '}
               {this.props.applicationData.meets_scholarship_criteria}
             </h4>
           )}
+
           {this.renderPointsSection()}
-          {this.props.applicationData.application_type ===
-            ApplicationTypes.teacher &&
-            this.props.applicationData.course === 'csp' && (
-              <h4>
-                <a
-                  target="_blank"
-                  href="https://drive.google.com/file/d/1_X_Tw3tVMSL2re_DcrSUC9Z5CH9js3Gd/view"
-                >
-                  View CS Principles Rubric
-                </a>
-              </h4>
-            )}
-          {this.props.applicationData.application_type ===
-            ApplicationTypes.teacher &&
-            this.props.applicationData.course === 'csd' && (
-              <h4>
-                <a
-                  target="_blank"
-                  href="https://drive.google.com/file/d/12Ntxq7TV1XYsD2eaZJVt5DqSctqR2hUj/view"
-                >
-                  View CS Discoveries Rubric
-                </a>
-              </h4>
-            )}
-          {this.props.applicationData.application_type ===
-            ApplicationTypes.facilitator && (
-            <h4>
-              <a
-                target="_blank"
-                href="https://docs.google.com/document/u/1/d/e/2PACX-1vTqUgsTTGeGMH0N1FTH2qPzQs1pVb8OWPf3lr1A0hzO9LyGLa27J9_Fsg4RG43ok1xbrCfQqKxBjNsk/pub"
-              >
-                View Rubric
-              </a>
-            </h4>
-          )}
+
+          <h4>
+            <a
+              target="_blank"
+              href={
+                this.props.applicationData.application_type ===
+                ApplicationTypes.teacher
+                  ? 'https://drive.google.com/file/d/1070Jf9VKtuJLOQJTCaO7fxUWyLOEHBdK/view'
+                  : 'https://docs.google.com/document/u/1/d/e/2PACX-1vTqUgsTTGeGMH0N1FTH2qPzQs1pVb8OWPf3lr1A0hzO9LyGLa27J9_Fsg4RG43ok1xbrCfQqKxBjNsk/pub'
```

Comment:
Maybe hoist this code up above to regular JS at the beginning of `renderHeader`, which sets a local variable, and then use the variable here?

Reply:
That looks cleaner. Will do